gtt/util.go

```diff
@@ -100,6 +100,9 @@ func match(result interface{}, expect interface{}) ([]string, interface{}, inter
 		}
 	case []interface{}:
 		if ra, ok := result.([]interface{}); ok {
+			if len(ra) != len(x) {
+				return []string{}, nil, expect
```

Comment:
Still not right, look again.
```
return []string{}, nil, v
```

Reply:
`v` is not defined until inside the for loop below?

Reply:
You are correct. Bigger change needed. Here is what I have.

```go
			for i, v := range x {
				if len(ra) <= i {
					return []string{}, nil, v
				}
				if loc, av, xv := match(ra[i], v); loc != nil {
					return append([]string{strconv.Itoa(i)}, loc...), av, xv
				}
			}
```